src/services/cli-http-client.js

```diff
@@ -17,6 +18,14 @@ class CliRequestClient {
     this.commandName = commandName;
     this.logger = logger;
     this.http = http || require('axios');
+    if (process.env.HTTP_PROXY) {
+      /*
+       * If environment variable HTTP_PROXY is set,
+       * add an appropriate httpsAgent to axios.
+       */
+      this.http.defaults.proxy = false;
```

Comment:
Why does this need to be set to false?

Reply:
If not explicitly `false`, axios will create a proxy given there is an `http_proxy` env variable set ([line 125](https://github.com/axios/axios/blob/fe52a611efe756328a93709bbf5265756275d70d/lib/adapters/http.js#L125) onwards of their http.js adapter). This proxy will overshadow the one set in the custom http agent and it does not work.

Axios' proxy not working has been reported in a few of their issues such as [here](https://github.com/axios/axios/issues/2072) and [here](https://github.com/axios/axios/issues/925).